scopus/classes/search.py

```diff
@@ -76,24 +80,45 @@ def __init__(self, query, api, refresh, count=200, start=0,
                 self._json = [loads(line) for line in f.readlines()]
         else:
             # Get a count of how many things to retrieve from first chunk
-            params = {'query': query, 'count': count, 'start': 0, 'view': view}
-            res = download(url=URL[api], params=params, accept="json").json()
-            n = int(res['search-results'].get('opensearch:totalResults', 0))
-            if n > max_entries:  # Stop if there are too many results
-                text = ('Found {} matches. Set max_entries to a higher '
-                        'number or change your query ({})'.format(n, query))
-                raise ScopusQueryError(text)
-            self._json = res.get('search-results', {}).get('entry', [])
-            if n == 0:
-                self._json = ""
-            # Download the remaining information in chunks
-            while n > 0:
-                n -= count
-                start += count
-                params.update({'count': count, 'start': start})
+
+            if cursor == False:
+                params = {'query': query, 'count': count, 'start': 0, 'view': view}
                 res = download(url=URL[api], params=params, accept="json").json()
-                self._json.extend(res.get('search-results', {}).get('entry', []))
-            # Finally write out the file
-            with open(qfile, 'wb') as f:
-                for item in self._json:
-                    f.write('{}\n'.format(dumps(item)).encode('utf-8'))
+                n = int(res['search-results'].get('opensearch:totalResults', 0))
+                if n > max_entries:  # Stop if there are too many results
+                    text = ('Found {} matches. Set max_entries to a higher '
+                            'number, change your query ({}) or set cursor=True'.format(n, query))
+                    raise ScopusQueryError(text)
+                self._json = res.get('search-results', {}).get('entry', [])
+                if n == 0:
+                    self._json = ""
+                # Download the remaining information in chunks
+                while n > 0:
+                    n -= count
+                    params.update({'count': count, 'start': start})
+                    print(params)
+                    res = download(url=URL[api], params=params, accept="json").json()
+                    self._json.extend(res.get('search-results', {}).get('entry', []))
+                # Finally write out the file
+                with open(qfile, 'wb') as f:
+                    for item in self._json:
+                        f.write('{}\n'.format(dumps(item)).encode('utf-8'))
+            else:
+                params = {'query': query, 'count': count, 'cursor': '*', 'view': view} # see https://dev.elsevier.com/tecdoc_developer_faq.html
```

Comment:
This and line 85 can be combined: Somewhere before the new line 84, create the `params` dict with items used in both cases. Then in line 85 (old), update `params` with the count. Then in this line, update `params` with items used for parsing with the cursor.